test/parallel/test-stream-some-every.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const common = require('../common');
+const {
+  Readable,
+} = require('stream');
+const assert = require('assert');
+
+function oneTo5() {
+  return Readable.from([1, 2, 3, 4, 5]);
+}
+
+function oneTo5Async() {
+  return oneTo5().map(async (x) => {
+    await Promise.resolve();
+    return x;
+  });
+}
+{
+  // Some and every work with a synchronous stream and predicate
```

Comment:
```suggestion
  // Some and every works with a synchronous stream and predicate
```

Reply:
This should likely be fixed the other way around I think